pkg/tarutil/tar.go

```diff
@@ -26,6 +27,10 @@ type Opts struct {
 	// to include all sub-directories. Set to true to prevent this
 	// behavior.
 	NoRecursion bool
+
+	// Change to this directory before any operations. Generated tar file
```

Comment:
nit: may need a bit format change to start the comment with `ChangeDirectory`, so internal presubmits won't be mad

// ChangeDirectory ...

Reply:
Fix this and I'll +2 ... we need to get  this in to remedy the CVE.